docs/libcurl/opts/CURLOPT_SSL_CTX_DATA.3

```diff
@@ -112,8 +112,8 @@ int main(void)
 }
 .fi
 .SH AVAILABILITY
-Added in 7.11.0 for OpenSSL. Added in 7.42.0 for wolfSSL/CyaSSL. Other SSL
-backends not supported.
+Added in 7.11.0 for OpenSSL. Added in 7.42.0 for wolfSSL. Other SSL backends
+not supported.
```

Comment:
Should we reword this while at it?
> Other SSL backends do not support this option.

Reply:
any particular suggestion?

Reply:
How about:

Added in 7.11.0 for OpenSSL and 7.42.0 for wolfSSL. Other SSL backends are
not supported.